code/modules/organs/organ_icon.dm

```diff
@@ -235,7 +235,7 @@ var/global/list/limb_icon_cache = list()
 		//VOREStation Edit End
 
 	// Translucency.
-	if(nonsolid) applying += rgb(,,,180) // SO INTUITIVE TY BYOND
+	if(transparent) applying += rgb(,,,180) // SO INTUITIVE TY BYOND //VOREStation Edit
```

Comment:
Why switch to this? Does nonsolid do anything else?

Reply:
As far as I can see, nonsolid is a factor in how slime limbs are removed. I went ahead and made a new "transparent" variable to prevent from messing up how slime limbs drop. (See: https://github.com/VOREStation/VOREStation/blob/master/code/modules/organs/organ_external.dm#L357)

I'm not really certain how it'd exactly mess up slime limbs dropping, but from the looks of it, it'd cause them to drop a proper "limb" if their nonsolid was set to 0 and they took anything but brute damage, instead of exploding into gibs like it currently does.